src/sentry/static/sentry/app/views/eventsV2/table/tableModalEditColumn.tsx

```diff
@@ -192,15 +193,11 @@ class TableModalEditColumnBodyForm extends React.Component<
   }
 }
 
-// TODO(leedongwei): Check with Mimi to get the link for DiscoverV2 docs
 const TableModalEditColumnFooter = () => (
   <FooterContent href={`${DISCOVER2_DOCS_URL}query-builder/`} target="_blank">
-    <div>
-      <InlineSvg src="icon-docs" /> {t('Read the docs')}
-    </div>
-    <div>
-      <InlineSvg src="icon-chevron-right" />
-    </div>
+    <IconDocs />
+    &nbsp;{t('Read the docs')}
```

Comment:
Might make sense to use a `styled(IconDocs)` with a `margin-right` instead of using `&nbsp;`